distrobox-create

```diff
@@ -219,6 +221,13 @@ while :; do
 				shift
 			fi
 			;;
+		--pre-init-hooks)
+			if [ -n "$2" ]; then
+				container_pre_init_hook="$2"
```

Comment:
we can define it like
`"container_pre_init_hook="--pre-init-hooks \"${2}\""

So later we won't need another variable